ingestion/create_cadet_databases_source/source.py

```diff
@@ -80,20 +84,35 @@ def _get_domains(self, manifest) -> set[str]:
             if manifest["nodes"][node]["resource_type"] == "model"
         )
 
-    def _get_databases_with_domains(self, manifest) -> list[tuple[str, str]]:
+    def _get_databases_with_domains_and_display_tags(
+        self, manifest
+    ) -> tuple[set[tuple[str, str]], dict]:
         """
         These mappings will only work with tables named {database}__{table}
-        like create a derived table
+        like create a derived table.
+
+        returns a set of databases with associated domain and a dict for
+        display tags, where key is database and value is dc_display_in_catalogue
+        if any model is to be displayed
         """
         mappings = set()
+        tags = {}
         for node in manifest["nodes"]:
             if manifest["nodes"][node]["resource_type"] == "model":
                 fqn = manifest["nodes"][node]["fqn"]
                 if validate_fqn(fqn):
                     database = fqn[-1].split("__")[0]
                     domain = fqn[1]
+                    tag = (
+                        "dc_display_in_catalogue"
+                        if "dc_display_in_catalogue" in manifest["nodes"][node]["tags"]
+                        else None
+                    )
                     mappings.add((database, domain))
-        return mappings
+                    if tag is not None:
+                        tags[database] = tag
```

Comment:
[Minor] If you make this value a list here, you can use it directly on line 73 without a conditional